conf/base.config

```diff
@@ -67,12 +64,8 @@ process {
   withName: alpha_diversity {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
```

Comment:
same as above, is this needed?

Reply:
This one probably is not really needed, I have never experienced it failing.